.gitignore

```diff
@@ -6,6 +6,7 @@ coverage/
 # Built assets created at npm install/prepublish time
 # See https://docs.npmjs.com/misc/scripts
 client/fonts/
-client/js/libs.min.js.map
-client/js/libs.min.js
-client/js/lounge.templates.js
+client/js/bundle.js
+client/js/bundle.js.map
+client/js/bundle.vendor.js
+client/js/bundle.vendor.js.map
```

Comment:
I think we can ignore vendor bundle map. Map for our own code is useful to have.